cmd/gsmpc-client/main.go

```diff
@@ -405,12 +410,23 @@ func reqKeyGen() {
 	fmt.Printf("smpc_getReqAddrNonce = %s\nNonce = %d\n", reqAddrNonce, nonce)
 	// build Sigs list parameter
 	sigs := ""
-	if *mode == "0" || *mode == "2" {
+	//if *mode == "0" || *mode == "2" {
+	if *mode == "0" {
 		for i := 0; i < len(enodesSig)-1; i++ {
 			sigs = sigs + enodesSig[i] + "|"
 		}
 		sigs = sigs + enodesSig[len(enodesSig)-1]
 	}
+
+	if *mode == "2" {
```

Comment:
please update the code: mode = 2 to mode = SHARE, and same for other number, better to understand.

Reply:
okay